docs/DeveloperGuide.md

```diff
@@ -28,7 +42,7 @@ command.
 
 The following shows the UML diagram used for the parser component implemented in MyLedger.
 
-![](team/images/parserOverview.png)
+![](team/images/ParserOverview.png)
```

Comment:
Changing the naming of the image without changing the name of the corresponding image file will result in a wrong path, maybe can just keep it?